content/devs/boone-raye-flores.json

```diff
@@ -26,8 +27,8 @@
       },
       {
         "type": "website",
-        "url": "http://www.booneraye.online/"
+        "url": "https://traveltayo.com"
```

Comment:
I'd be happy to merge your PR, but just as a friendly reminder, this is a Dev Board. This might not be the best place to promote your Travel Facebook page. Thank you!